qml.qrc

```diff
@@ -448,5 +448,7 @@
         <file>atomic_defi_design/qml/Wallet/Wallet.qml</file>
         <file>atomic_defi_design/qml/main.qml</file>
         <file>qtquickcontrols2.conf</file>
+        <file>atomic_defi_design/qml/Portfolio/PieItem.qml</file>
+        <file>atomic_defi_design/qml/Components/DexFadebehavior.qml</file>
```

Comment:
qrc file is now sorted, please respect the sort order

Reply:
Qt creator add dynamicaly new line in qrc